README.md

```diff
@@ -108,7 +108,8 @@ POST /token
 
 ```json
```

Comment:
I think we agreed to add `create_room` to JSON POST payload that allows user to skip creating the room right?

Reply:
Oh right. Was it `adhoc: true` that would skip room creation?